core/server/proxy/src/main/java/alluxio/proxy/s3/ListPartsResult.java

```diff
@@ -0,0 +1,261 @@
+  public boolean isTruncated() {
+    return mIsTruncated;
+  }
+
+  /**
+   * @param isTruncated whether the results are truncated
+   */
+  @JacksonXmlProperty(localName = "IsTruncated")
+  public void setIsTruncated(boolean isTruncated) {
+    mIsTruncated = isTruncated;
+  }
+
+  /**
+   * @return the list of parts
+   */
+  @JacksonXmlProperty(localName = "Part")
+  @JacksonXmlElementWrapper(useWrapping = false)
+  public List<Part> getParts() {
+    return mParts;
+  }
+
+  /**
+   * @param parts the list of parts to set
+   */
+  @JacksonXmlProperty(localName = "Part")
+  @JacksonXmlElementWrapper(useWrapping = false)
+  public void setParts(List<Part> parts) {
+    mParts = parts;
+  }
+
+  /**
+   * Part contains metadata of a part of the object in multipart upload.
+   */
+  @JsonPropertyOrder({ "PartNumber", "LastModified", "ETag", "Size" })
+  public static class Part {
+    // Part number.
+    private int mPartNumber;
+    // Last modification time of the part.
+    private String mLastModified;
+    // Entity tag of the part.
+    private String mETag;
+    // Size of the part in bytes.
+    private long mSize;
+
+    /**
+     * @param status the status of the part file
+     * @return the {@link Part} parsed from the status
+     */
+    public static Part fromURIStatus(URIStatus status) {
+      Part result = new Part();
+      result.setPartNumber(Integer.parseInt(status.getName()));
+      result.setLastModified(S3RestUtils.toS3Date(status.getLastModificationTimeMs()));
+      result.setSize(status.getLength());
+      return result;
+    }
+
+    /**
+     * Constructs a default {@link Part}.
+     */
+    public Part() {
+      mPartNumber = 0;
+      mLastModified = "";
+      mETag = "\"\"";
+      mSize = 0;
+    }
+
+    /**
+     * @return the part number
+     */
+    @JacksonXmlProperty(localName = "PartNumber")
+    public int getPartNumber() {
+      return mPartNumber;
+    }
+
+    /**
+     * @param partNumber the part number to set
+     */
+    @JacksonXmlProperty(localName = "PartNumber")
+    public void setPartNumber(int partNumber) {
+      mPartNumber = partNumber;
+    }
+
+    /**
+     * @return the last modification time
+     */
+    @JacksonXmlProperty(localName = "LastModified")
+    public String getLastModified() {
+      return mLastModified;
+    }
+
+    /**
+     * @param lastModified the last modification time to set
+     */
+    @JacksonXmlProperty(localName = "LastModified")
+    public void setLastModified(String lastModified) {
+      mLastModified = lastModified;
+    }
+
+    /**
+     * @return the entity tag surrounded by quotes
+     */
+    @JacksonXmlProperty(localName = "ETag")
+    public String getETag() {
+      return mETag;
+    }
+
+    /**
+     * @param etag the entity tag to set (without surrounding quotes)
+     */
+    @JacksonXmlProperty(localName = "ETag")
+    public void setETag(String etag) {
+      mETag = "\"" + etag + "\"";
```

Comment:
extract `"\"" + etag + "\""` as a helper function in `S3RestUtils` ? it's used for a couple of times

Reply:
done